Jenkinsfile

```diff
@@ -121,7 +121,7 @@ pipeline {
 
       }
       steps {
-        sh 'docker run -d -p 8090:8090 -e DATABASE_HOST=172.18.0.4:5432 -e DATABASE_NAME -e DATABASE_USER -e DATABASE_PASSWORD -e APP_SECRET --name backendrun --restart always --net netapp -it backend'
+        sh 'docker run -d -p 8090:8090 -e DATABASE_HOST=172.18.0.4:5432 -e DATABASE_NAME -e DATABASE_USER -e DATABASE_PASSWORD -e APP_SECRET --name STAGE='PROD' backendrun --restart always --net netapp -it backend'
```

Comment:
That's not right, notice you are changing value of the `--name` argument. Should be `-e STAGE='PRO'`

Reply:
Still not quite right `-e STAGE=STA --name backendruntest`